conda/install.py

```diff
@@ -1009,14 +1011,17 @@ def move_path_to_trash(path):
         trash_file = tempfile.mktemp(dir=trash_dir)
 
         try:
-            shutil.move(path, trash_file)
+            os.rename(path, trash_file)
         except OSError as e:
             log.debug("Could not move %s to %s (%s)" % (path, trash_file, e))
         else:
+            log.debug("Moved to trash: %s" % (path,))
             delete_linked_data_any(path)
-            return trash_file
+            if not preclean:
+                rm_rf(trash_file, max_retries=1, trash=False)
+            return True
 
-    log.debug("Could not move %s to trash" % path)
+    return None
```

Comment:
Originally I had this return the path to the moved file. I'll fix to avoid confusion.